.pre-commit-config.yaml

```diff
@@ -284,4 +284,4 @@ repos:
       - id: oxipng
         name: run oxipng
         description: check PNG files with oxipng
-        args: ['-o', '4', '--strip', 'safe', '--alpha']
+        args: ['--fix', '-o', '4', '--strip', 'safe', '--alpha']
```

Comment:
What is the benefit of `--fix`?

Reply:
> What is the benefit of `--fix`?

```
--fix
          Do not perform checksum validation of PNG chunks. This may allow some files with errors to
          be processed successfully.
```

https://github.com/oxipng/oxipng/blob/1c02ee92996297bf9015c462a85d9770c97d4c06/MANUAL.txt#L160

Reply:
refs https://github.com/brisbanesocialchess/brisbanesocialchess.github.io/pull/168#pullrequestreview-2999468704